src/main/java/nextstep/ladder/model/Ladder.java

```diff
@@ -0,0 +1,35 @@
+package nextstep.ladder.model;
+
+import nextstep.ladder.model.value.Line;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Random;
+import java.util.stream.IntStream;
+
+public class Ladder {
+
+    private static final int START_NUMBER = 0;
+    private static Random RANDOM = new Random();
+
+    private static List<Line> lines = new ArrayList<>();
+
+    private Ladder(List<Line> ladder) {
+        this.lines = ladder;
```

Comment:
this는 생략해도 될 것 같습니다! 
```suggestion
        lines = ladder;
```